pennylane/templates/subroutines/adder.py

```diff
@@ -22,54 +22,83 @@
 class Adder(Operation):
     r"""Performs the in-place modular addition operation.
 
-     This operator performs the modular addition by an integer :math:`k` modulo :math:`mod` in the
-     computational basis:
+    This operator performs the modular addition by an integer :math:`k` modulo :math:`mod` in the
+    computational basis:
 
-     .. math::
+    .. math::
 
-         \text{Adder}(k, mod) |x \rangle = | x+k \; \text{modulo} \; mod \rangle.
+        \text{Adder}(k, mod) |x \rangle = | x+k \; (mod) \rangle.
 
-     The implementation is based on the quantum Fourier transform method presented in
-     `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
+    This operation can be represented in a quantum circuit as:
+
+    .. figure:: ../../_static/templates/arithmetic/adder.png
+        :align: center
+        :width: 60%
+        :target: javascript:void(0);
+
+    The implementation is based on the quantum Fourier transform method presented in
+    `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
 
     .. note::
 
-        Note that :math:`x` must be smaller than :math:`mod` to get the correct result.
+        :math:`x` must be smaller than :math:`mod` to get the correct result.
```

Comment:
I see that "Note that" sounded repititive here but starting with a math symbol, x, is also not ideal.

Reply:
I've changed to: "To obtain the correct result, :math:`x` must be smaller than :math:`mod`"